src/spacingData.js

```diff
@@ -1,22 +1,41 @@
+// @flow
 /**
  * Describes spaces between different classes of atoms.
  */
 
-const thinspace = {
+export type Space = {| number: number, unit: "mu" |};
+
+const thinspace: Space = {
     number: 3,
     unit: "mu",
 };
-const mediumspace = {
+const mediumspace: Space = {
     number: 4,
     unit: "mu",
 };
-const thickspace = {
+const thickspace: Space = {
     number: 5,
     unit: "mu",
 };
 
+// Making the type below exact with all optional fields doesn't work due to
+// - https://github.com/facebook/flow/issues/4582
+// - https://github.com/facebook/flow/issues/5688
+// However, since *all* fields are optional, $Shape<> works as suggested in 5688
+// above.
```

Comment:
Interesting.  Thanks for adding links back to the issues.

Reply:
I want to add one thing before this can be submitted. Per the discussion at https://github.com/facebook/flow/issues/4582, I need to add a `& {}` to the `$Shape<>` type since `$Shape<>` seems to currently allow `null|undefined`. Feel free to do this yourself before merging (if everything else is LG). Otherwise, I'll do it tonight,

Reply:
Done fixed. It's ready now.